src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Data;
+import mate.academy.spring.model.Ticket;
+
+@Data
+public class OrderResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
```

Comment:
You can return information about tickets in your ShoppingCartResponseDto / OrderResponseDto in the form of List<Long> tiketIds (checklist)